edith-step/src/App.css

```diff
@@ -1,38 +1,52 @@
-.App {
-  text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.background-image-0 {
+  background-attachment: fixed;
+  background-image: url(./receipt.jpg);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 150vh;
+  position: relative;
+  width: 100vw;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.app-describe {
+  color: #000;
+  left: 0;
+  opacity: 0.8;
+  position: absolute;
+  text-align: center;
+  top: 50%;
+  width: 100%;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+.border {
+  border: none !important;
+  color: #fff;
+  font-size: 30px;
+  letter-spacing: 10px;
+  padding: 15px;
 }
 
-.App-link {
-  color: #61dafb;
+.scroll-down {
+  animation-delay: 1s;
+  animation-direction: alternate;
+  animation-duration: 1s;
+  animation-iteration-count: infinite;
+  animation-name: scroll-down;
+  animation-timing-function: ease-in-out;
```

Comment:
> Use shorthand properties where possible [...] even in cases where only one value is explicitly set.

go/htmlcssstyle#Shorthand_Properties

```css
animation: 1s ease-in-out 1s infinite alternate scroll-down;
```

Reply:
Alternate URL: https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties

See https://developer.mozilla.org/en-US/docs/Web/CSS/animation

Reply:
Thanks for the reference. Changes made.